src/test/java/co/omise/OmiseTest.java

```diff
@@ -1,14 +1,24 @@
 package co.omise;
 
-import com.google.common.io.ByteStreams;
 import org.junit.Assert;
 
+import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 
 public abstract class OmiseTest extends Assert {
     public byte[] getResourceBytes(String path) throws IOException {
         InputStream stream = getClass().getResourceAsStream(path);
-        return stream == null ? null : ByteStreams.toByteArray(stream);
+        return stream == null ? null : InputStreamToByteArray(stream);
     }
-}
+
+    private static byte[] InputStreamToByteArray(InputStream input) throws IOException {
```

Comment:
Could you change to camel-case?

Reply:
We're going to bring back all the Guava methods used in tests, so this will be removed anyway